src/components/MaplibreMap.jsx

```diff
@@ -317,9 +312,9 @@ const MaplibreMap = ({ view, setView }) => {
         ref={mapRef}
         style={{ width: '100%', height: '100%' }}
         initialViewState={{
-          longitude: (mapCenter && mapCenter[1]) || defaultLon,
```

Comment:
`0` are all legal values for lat, lng, and zoom, so converted to nullish coallescing instead of logical or.  Also opted for optional chaining to avoid checking if the coordinate array is defined before reading from it.